TestCases/nonequilibrium/thermalbath/finitechemistry/weakly_ionized.cfg

```diff
@@ -0,0 +1,92 @@
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%                                                                              %
+% SU2 configuration file                                                       %
+% Case description: 2D 5x5 Square Test Case representing a thermal bath        %
+%                    with thermal nonequilibrium and finite-rate chemistry     %
+%                                                                              %
+%                   This is only serves as a validation case.                  %
+%                                                                              %
+%                                                                              %
+% Author: W. Maier                                                             %
+% File Version 7.3.0 "Blackbird"                                               %
+%                                                                              %
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+% ------------- DIRECT, ADJOINT, AND LINEARIZED PROBLEM DEFINITION ------------%
+%
+% Physical governing equations (EULER, NAVIER_STOKES,
+%                               WAVE_EQUATION, HEAT_EQUATION, FEM_ELASTICITY,
+%                               POISSON_EQUATION)
+SOLVER= NEMO_EULER         
+GAS_MODEL= AIR-7
+GAS_COMPOSITION=(0.0, 0.767, 0.233, 0.0, 0.0, 0.0, 0.0)
+MATH_PROBLEM= DIRECT
+READ_BINARY_RESTART= NO
+
+TIME_MARCHING=TIME_STEPPING
+TIME_DOMAIN= YES
+TIME_STEP= 0.000000001 % 1.e-09
+MAX_TIME= 0.001
+
+% ----------- COMPRESSIBLE AND INCOMPRESSIBLE FREE_STREAM DEFINITION ----------%
+%
+MACH_NUMBER= 0.0
+AOA = 0.0
+FREESTREAM_PRESSURE = 101325.0
+FREESTREAM_TEMPERATURE = 30000
+FREESTREAM_TEMPERATURE_VE= 1000 
+
+% ---- NONEQUILIBRIUM GAS, IDEAL GAS, POLYTROPIC, VAN DER WAALS AND PENG ROBINSON CONSTANTS -------%
+FLUID_MODEL= SU2_NONEQ
+
+% -------------------- BOUNDARY CONDITION DEFINITION --------------------------%
+%
+MARKER_SYM= (left,bottom,top,right )
+
+% ------------- COMMON PARAMETERS DEFINING THE NUMERICAL METHOD ---------------%
+%
+NUM_METHOD_GRAD= WEIGHTED_LEAST_SQUARES
+CFL_NUMBER= 1.0
+TIME_ITER= 11
+
+% ------------------------ LINEAR SOLVER DEFINITION ---------------------------%
+%
+LINEAR_SOLVER= FGMRES
+LINEAR_SOLVER_PREC= ILU
+LINEAR_SOLVER_ERROR= 1E-10
+LINEAR_SOLVER_ITER= 10
+
+% -------------------- FLOW NUMERICAL METHOD DEFINITION -----------------------%
+%
+CONV_NUM_METHOD_FLOW= AUSM
+MUSCL_FLOW= NO
+SLOPE_LIMITER_FLOW= VENKATAKRISHNAN
+TIME_DISCRE_FLOW= EULER_EXPLICIT
+
+% --------------------------- CONVERGENCE PARAMETERS --------------------------%
+%
+CONV_RESIDUAL_MINVAL= -50
+CONV_STARTITER= 10
+CONV_CAUCHY_ELEMS= 100
+CONV_CAUCHY_EPS= 1E-6
+OBJECTIVE_FUNCTION= DRAG
+
+% ------------------------- INPUT/OUTPUT INFORMATION --------------------------%
+%
```

Comment:
It would be good to include some coefficient outputs (lift etc.) in the regressions

Reply:
Good point!